services/catalog/openapi.json

```diff
@@ -363,7 +363,7 @@
             "required": true,
             "schema": {
               "title": "User Id",
-              "exclusiveMinimum": 0.0,
+              "exclusiveMinimum": false,
```

Comment:
this does not look good. this should be a number. something like ```0``` no ??

Reply:
Actually bool is the right type according to openapi standard. That is the reason why i had to override the ``FastAPI.openapi`` memeber. See [comments and links here](https://github.com/ITISFoundation/osparc-simcore/blob/74a3f3c19def0059df1a3260b8e8419ce2b81c76/packages/service-library/src/servicelib/fastapi/openapi.py#L46)

Reply:
hmm I don't think I agree with this.
this is an integer type. what does exclusiveMinimum: false mean?

Reply:
The following properties are taken directly from the JSON Schema definition and follow the same specifications:
- exclusiveMinimum

Reply:
Ah, I noticed that i missed one link. Check this

https://github.com/tiangolo/fastapi/issues/240

Reply:
Indeed ... the patch i apply is not complete. But now it should be. 

See https://github.com/ITISFoundation/osparc-simcore/blob/97470c3f5278b5d6aa577b17f591a01475900e09/packages/service-library/tests/fastapi/test_openapi.py#L24